arangod/Statistics/StatisticsFeature.cpp

```diff
@@ -90,6 +93,9 @@ class arangodb::StatisticsThread final : public Thread {
     int nothingHappened = 0;
 
     while (!isStopping() && StatisticsFeature::enabled()) {
+#ifdef ARANGODB_ENABLE_MAINTAINER_MODE
+      arangodb::lastStatisticsThreadActivity = TRI_microtime();
```

Comment:
That will give us many false positives if 
```
size_t count = RequestStatistics::processAll();
```
will take more than a second.
I suggest to move   this to line 112
```
std::this_thread::sleep_for(std::chrono::microseconds(sleepTime));
```
and another before line  116
```
 if (count < 10) { 
```
And after sleep reset variable to some "not sleeping" value, say -1
So we could certainly know if thread is sleeping (and how long) or it is just working on some statistics